app/models/katello/content_view_version.rb

```diff
@@ -167,6 +167,10 @@ def promote_puppet_environment?
       (!content_counts.blank? && content_counts.dig(PuppetModule::CONTENT_TYPE) > 0) || self.content_view.force_puppet_environment?
     end
 
+    def puppet_module_count
+      content_counts.dig(PuppetModule::CONTENT_TYPE) if content_counts?
```

Comment:
This line can be written as:
```
content_counts&.dig(PuppetModule::CONTENT_TYPE)
```

Where is `puppet_module_count` get called?

Reply:
@lfu I will fix that up, here is where we are looking:

https://github.com/Katello/katello/blob/KATELLO-3.18/engines/bastion_katello/app/assets/javascripts/bastion_katello/content-views/details/components/views/content-view-composite-available-content-views.html#L77

Reply:
@lfu updated